tests/test_entities.py

```diff
@@ -344,6 +345,7 @@ def test_id_and_which(self):
             (entities.Product, 'sync'),
             (entities.PuppetClass, 'smart_class_parameters'),
             (entities.Repository, 'docker_manifests'),
+            (entities.Repository, 'delete_with_args'),
```

Comment:
This is failing because it isn't included in the entity's which tuple. Likely doesn't need to be added here since it isn't a true path.